### PR TITLE
Port colorconverter YCbCr and YCCk  to arm

### PR DESCRIPTION
**Pull request by stefannikolei**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [x] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Benchmark:
YCbCr
|        Method |       Mean |    Error |   StdDev |
|-------------- |-----------:|---------:|---------:|
|        Scalar | 4,777.9 ns | 15.98 ns | 14.95 ns |
|   SimdVector8 |   295.5 ns |  1.07 ns |  1.00 ns |
| SimdVectorAvx |         NA |       NA |       NA |
| SimdVectorArm |   188.3 ns |  0.87 ns |  0.77 ns |

YCCk
|          Method |       Mean |    Error |   StdDev | Ratio | RatioSD |
|---------------- |-----------:|---------:|---------:|------:|--------:|
|          Scalar | 5,934.5 ns | 12.66 ns | 11.22 ns |  1.00 |    0.00 |
|     SimdVector8 |   327.8 ns |  0.74 ns |  0.70 ns |  0.06 |    0.00 |
|  SimdVectorAvx2 |         NA |       NA |       NA |     ? |       ? |
| SimdVectorArm64 |   228.3 ns |  0.47 ns |  0.42 ns |  0.04 |    0.00 |

### CODE REVIEW

**Comment by stefannikolei** [comment]
With this all Jpeg ColorConverters have ARM specific code

**Comment by antonfirsov** [comment]
This PR has to adapt the test changes from #2416 now.

File: tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs [hunk 1]
```diff
@@ -191,7 +191,7 @@ static void RunTest(string arg)
             }
             else if (AdvSimd.IsSupported)
```
Comment (antonfirsov) [inline tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs:192]:
```suggestion
            else if (AdvSimd.Arm64.IsSupported)
```

Same for Cmyk.

File: tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs [hunk 2]
```diff
@@ -536,6 +553,23 @@ public void FromYccKVector(int seed)
             new JpegColorConverterBase.YccKScalar(8),
             precísion: 4);
 
+    [Theory]
+    [MemberData(nameof(Seeds))]
+    public void FromYccKArm64(int seed) =>
+        this.TestConversionToRgb( new JpegColorConverterBase.YccKArm64(8),
```
Comment (antonfirsov) [inline tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs:559]:
```suggestion
        this.TestConversionToRgb(new JpegColorConverterBase.YccKArm64(8),
```

**Review by antonfirsov — APPROVED** [review]
:shipit:

**Pull request merged**